main.js

```diff
@@ -6,9 +6,23 @@ const createWindow = () => {
     height: 600,
   });
 
+  // uncomment this to unlock chrome devtools
+  // win.webContents.openDevTools()
```

Comment:
I don't how is work in desktop applications, but most likely the same. We have to create file of environment, usually it calls .env but I'm not sure about this framework. Consequently, depends on environment we have a few files for production and local development. These files have a variables with configurations. Relate to this comment, it will variable with `APP_ENV=production` or `APP_ENV=local`
If you are note familiar with 12-factor applications, I recommend to deep into it [here](https://12factor.net/) and read some about in relate to heroko.

Reply:
@agoalofalife thanks for the link, makes sense, but i do not know how to manage these environment variables. do maybe have any tutorials or videos to check out?

Reply:
To be brutally honest I don't know right way in electron. I found a few links how to do it in node.js and you just follow them and will find appropriate convention for electron.
https://www.twilio.com/en-us/blog/environment-variables-env-file-node
https://philna.sh/blog/2023/09/05/nodejs-supports-dotenv/

and some already predefined in electron
https://www.electronjs.org/docs/latest/api/environment-variables

basic script to create some env file and define variables which are depend on environment

Reply:
@agoalofalife  now the environment type is managed from the .env, thanks!